bot/exts/recruitment/talentpool/_cog.py

```diff
@@ -406,9 +406,11 @@ async def _edit_nomination_reason(
             return
         if isinstance(target, int):
             nomination_id = target
+            target_mention = (await self.bot.fetch_user(target)).mention
```

Comment:
When `target` is an int, it is a nomination_id, not a user_id, so this will never work.
it might be easier to do this using the `nomination` variable, that is the response from `await self.bot.api_client.get(f"bot/nominations/{nomination_id}")`. Since that returns a [nomination object](https://github.com/python-discord/site/blob/main/pydis_site/apps/api/models/bot/nomination.py#L7-L37), which contains a [User object](https://github.com/python-discord/site/blob/ecf794b87a5fa8026e5cfd2f996a4ade143a9042/pydis_site/apps/api/models/bot/user.py#L18) as one of it's keys

Reply:
I see, thanks!

Reply:
I grabbed `nomination['user']` and then ran `await self.bot.fetch_user` given the ID, and it seems to work well now.

One quick question - I know that fetch_user launches an API request so would we want to pull the user from the cache first and if that fails, run an API request on the ID? or is the user object already existing within the nomination object, specifically `nomination['user']`

Reply:
Is there a need to fetch the user at all? `nomination['user']` should have the id in it, so you can just do <@id> manually.

Reply:
Oh wait, I forgot how mentions were formatted in the first place. Thanks!